wpengine/faustjs
# [Merl 906] Added CoreHeading block implementation

## Tasks


## Description



## Related Issue(s):



## Testing

* **USE TwoentyTwentyThree WP Theme.**
* Checkout the testing commit here, since it contains all the necessary code to get started https://github.com/wpengine/faustjs/pull/1472/commits/9dc1ba0a0c646b56d9dd3054e86e253a4f811977
* In the WordPress Admin, create different headings with different sizes and color combinations.
* Test that in the Decoupled site you can see the design matches with the WP Site with no major changes.


## Screenshots
<img width="1792" alt="Screenshot 2023-06-23 at 13 33 16" src="https://github.com/wpengine/faustjs/assets/328805/0f6f5f89-8da6-4c69-a90a-a1c9e779a4aa">



## Documentation Changes



## Dependant PRs
